content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -71,42 +111,76 @@ set(FILES
 )
 ```
 
+### Adding a temporary auto-component
+There's currently a [bug](https://github.com/o3de/o3de/issues/4058) causing a build failure if multiplayer auto-components are enabled, but no auto-components are created. As a work-around, create a temporary auto-component.
+1. Create a new folder under your project's `Code\Source\` directory called `AutoGen`. 
+    {{< note >}}This AutoGen directory doesn't have to be temporary; all multiplayer auto-components can live here.{{< /note >}}
+1. Create a new, temporary auto-component file under `Code\Source\AutoGen` called `MyFirstNetworkComponent.AutoComponent.xml`.
```

Comment:
Add note that name could be anything, its just important to ensure name is used consistently. For this guide we will use "MyFirstNetworkComponent" ?

Reply:
Good call out.